bin/mb

```diff
@@ -261,6 +261,7 @@ var aliases = {
         d: 'debug',
         i: 'allowInjection',
         l: 'localOnly',
+        si: 'saveImposters',
```

Comment:
I avoid 2-letter aliases, because it gets confusing on the command line. On the CLI, "mb -si" is usually equivalent to "mb -s -i", which would have a different meaning (e.g. -i is allowInjection). 

That said, this flag isn't necessary; we can infer whether we're starting in persistent mode simply by looking at the --savefile option, which makes the CLI easier to understand. See the next comment for how.

Reply:
We'll remove the alias.